ngraph_bridge/ie_basic_engine.cc

```diff
@@ -0,0 +1,80 @@
+/*******************************************************************************
```

Comment:
"Basic Engine" is unnecessary if you provide a default implementation for the abstract backend? Backends that don't need a custom "Infer" or custom configuration can fall back to the default impl.

Reply:
Is the basic execution should be in Executable or in BackendEngine? We would disagree with the first one. The second option might be possible and it may help with the issue about MYRIAD configurations above too. But we may need to think about it if it will impact anything else in the future. Although, we may change this design later, I don't think it's in a bad shape the be merged now.

Reply:
"Basic" execution should be in the implementation of the default Backend.